### ISSUE TITLE
tab pills without javascript

### ISSUE THREAD

**Issue by 1xch** [angular-ui/bootstrap#2037]
reference: https://github.com/angular-ui/bootstrap/issues/1871

Reopening this as a user who needs this project, but is looking to minimize javascript use in my projects, and my forced attention on javascript programming.

From what I'm getting is that:
1.  To get a class "nav nav-pills nav-stacked"
2.  $scope.navType = 'pills'; needs to be specified in a directive

This is mystifying behavior as I need to specify in my own jinja2 templates something along the lines of:

```
<tabset class={{ t.tabset_css_class }}{%if t.vertical%} vertical="true"{% endif %}{% if t.justified %} justified="true"{% endif %}>
```

[see](https://github.com/thrisp/flacro/blob/develop/flask_flacro/templates/macros/tab.html#L52-L62)

without resorting to a custom directive.

Just looking at:

https://github.com/angular-ui/bootstrap/blob/master/src/tabs/tabs.js#L80-L95

there should be a scope.pills in link: function

I KNOW that to js scripting and what you've done here, this is probably makes sense to you as why it is like it is, but from a user perspective, it represents a large detour for drop-in, immediate use of bootstrap css items.

(Not to mention that following this there is another issue of vertical tabs ONLY happening with nav-pills & not with nav-tabs which led to this issue, and if vertical tabs did work, this wouldn't be brought up at all but still be an existing issue...this could be my issue, but I can't get ease of use with the tabs as they are, and they've represented a hurdle to my usage of angular-ui, where angular-ui has been a breeze.)

Bottom line, the tabs fucntionality is lacking overall, tab usage doesn't provide a seamless interface to bootstrap css and I need to have control over bootstrap css without jumping through extra javascript hoops, but still make use of the angular-ui directives; please do not dismiss out of hand and consider what I'm saying and let me know what I do not fully understand, as stated I'm not a js programming and would like to minimize my use of js to the bare minimum.
